pygeo/geo_utils/tripan.py

```diff
@@ -0,0 +1,241 @@
+import numpy as np
+import os
+from .bilinear_map import getBiLinearMap
+
+# --------------------------------------------------------------
+#                Tripan-related class
+# --------------------------------------------------------------
```

Comment:
Should this be move to the same place as pygeo.py since both are used for geometry generation?

Reply:
I actually think this can be removed, since we deprecated Tripan I don't think anyone's using this code.

Reply:
I'll remove that file.